Support for sim line-of-sight tests via RPC

KerryMoffittRtn opened the pull request:
Includes method to obtain world extents

Commit:
Support for sim line-of-sight tests via RPC

Includes method to obtain world extents

zimmy87 reviewed (changes requested):
some comments on style

zimmy87 commented:
It seems that you have a lot of conflicts. This is most likely due to conflicts with autoformatted code in master. Please follow these steps to resolve this:

1. check out your pull request
2. run `git checkout master .clang-format` in your repo's root folder
3. if you don't have npm installed, download it from https://www.npmjs.com/get-npm
4. run `npm install -g clang-format`
5. run `clang-format -i path/to/file` for each conflicting file
6. run `git add -u` to track all style changes
7. run `git commit -am "apply style from clang-format"`
8. run `git merge master`
9. resolve other non-style conflicts if you have any
10. push changes

Thanks for helping us with these style changes!

zimmy87 committed:
apply style from clang-format

zimmy87 committed:
Merge branch 'master' into rpc-los-tests

zimmy87 committed:
fix build break

this addresses a few compile errors resulting from previous merge

zimmy87 committed:
various fixes

this commit fixes clang-formatting issues, a Unity build break, and a misnamed API in the docs
